### PR TITLE
Create LICENSE. Resolve #1

### LINKED ISSUE #1: Need licenses
We need an appropriate license for this repo, as well as maybe a way to include licenses for re-distributing the libraries that are being built by this project?

@Ingvord if you could help with this, it would be greatly appreciated.

### PR DESCRIPTION
**Pull request by Ingvord**
This adds Apache 2.0 license to this project

### CODE REVIEW

**Comment by Ingvord** [comment]
FYI @bmaranville

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (bmaranville) [inline LICENSE]: What should we put in here?

**Review by bmaranville — CHANGES_REQUESTED** [review]
There seems to be some information expected - not sure what we should put in for "copyright owner"

File: LICENSE (hunk 1 above)
Comment (Ingvord) [reply LICENSE]: @bmaranville good catch! yyyy has to be actual year, for the name I would suggest the name of the org

**Pull request merged**